code_schemes/s03e17.json

```diff
@@ -3,6 +3,94 @@
   "Name": "S03E17",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-583ed610",
+      "NumericValue": 0,
+      "CodeType": "Normal",
+      "DisplayText": "religion does not allow",
+      "StringValue": "religion_does_not_allow",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-583ed610",
+      "NumericValue": 1,
+      "CodeType": "Normal",
+      "DisplayText": "awareness creation",
+      "StringValue": "awareness_creation",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-583ed610",
+      "NumericValue": 2,
+      "CodeType": "Normal",
+      "DisplayText": "working together",
+      "StringValue": "working_together",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-583ed610",
+      "NumericValue": 3,
+      "CodeType": "Normal",
+      "DisplayText": "encouragement",
+      "StringValue": "encouragement",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-583ed610",
+      "NumericValue": 4,
+      "CodeType": "Normal",
+      "DisplayText": "give them their right",
```

Comment:
This is different to the string value (right vs. rights)

Reply:
Thanks for catching this.